routes/recaptcha_solver.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express')
+const router = express.Router()
+const utils = require('../helpers/utils')
+
+const DEFAULT_TIMEOUT = 1000;  // 1 second
+
+/*
+ * This module introduces new ability to puppeteer-service.
+ * It is capable of solving recaptchas on the given web-page.
+ * If there is no recaptcha on the page nothing bad will happen.
+ * If there is recaptcha it solves it and then inserts the special code
+ * into the page automatically.
+ */
+
+/**
+ *
+ * @param page - page with possible recaptcha.
+ * @param request - request to the page.
+ */
+
+async function action(page, request) {
+    const {
+        captchas,
+        filtered,
+        solutions,
+        solved,
+        error
+    } = await page.solveRecaptchas();
```

Comment:
We want such feedback as a first action in formResponse. Also we should have possibility to turn off captcha solving in request params.